pkg/sql/plan/function/func_unary.go

```diff
@@ -1055,6 +1055,153 @@ func octFloat[T constraints.Float](xs T) (types.Decimal128, error) {
 	return res, nil
 }
 
+// data structs for encoding and decoding
```

Comment:
You are not going to implement your own crypto function.   Use AES.

Reply:
Hi fengttt, according to the issue https://github.com/matrixorigin/matrixone/issues/15027, my implementation is aligned with the method used by MySQL for their ENCODE() and DECODE() functions. MySQL utilizes a simple pseudo-random number generator based algorithm for encoding and decoding data, which is designed for quick and straightforward operations rather than strong security like AES.

Reply:
> Hi fengttt, according to the issue #15027, my implementation is aligned with the method used by MySQL for their ENCODE() and DECODE() functions. MySQL utilizes a simple pseudo-random number generator based algorithm for encoding and decoding data, which is designed for quick and straightforward operations rather than strong security like AES.

No, we don't care what MySQL does.   It is broken.   Use a real one crypto strength algorithm.